data/geolocation-cn

```diff
@@ -820,11 +820,15 @@ linkvans.com
 linsn.com
 linuxidc.com
 liulanqi.net
+liulishuo.com
```

Comment:
I would prefer to put these domains together, like this:
```
# 流利说
liulishuo.com
llsapp.com
llscdn.com
llsserver.com

...(other domains that previously exist in this category)
```

Reply:
done